### ISSUE TITLE
Allow READMEs inside of theme Directories to allow for better organization of theme subrepositories.

### ISSUE THREAD

**Issue by joamjoamjoam** [suchmememanyskill/CssLoader-ThemeDb#54]
Example Theme with Readme:

[Gawr Gura](https://github.com/joamjoamjoam/SteamDeckThemes/tree/main/Gawr%20Gura)

As a theme developer I would like to provide extra pictures and explanations for my themes to potential users so that it is clear what my theme does.

As a theme user I find that the single picture allowed on the store page is not enough to explain what a theme does. An example of this is No Game Shine where the changes are subtle and with only a picture with no description it's hard to understand what it does. If you navigate to the no Game Shine sub repo there is no explanation there and what it does either.

It would be beneficial from both viewpoints to allow READMEs per themes to allow developers to explain to their users what their theme does. It would extra beneficial to require a description or better render the Readme on the Theme Page in the Decky Store

**Comment by PartyWumpus** [comment]
Currently this is possible if you put the theme inside of a directory in your repo and place the readme outside the folder. You also need to update the .json here to reflect the change. Obviously it's not super convenient, but possible. Example: [my theme repo](https://github.com/vrman123/PoyomannSteamDeckThemes)

Anyway, with the new description feature, I think this is mostly made pointless but support for multiple images and possibly video support inside of the 'view details' menu would probably be enough to completely replace readme files and would much more convenient for end users.  If necessary I can open a separate issue requesting this feature, as there's no reason there can't be both.

Just my 2 cents.

**Comment by suchmememanyskill** [comment]
So what exactly is this issue about?
- ignoring readme files? (Or any other files)
- making the readme files show up in the ui?
- adding multiple images?

**Comment by joamjoamjoam** [comment]
Mainly the lack of a way to describe what the theme does with more than just a single picture. Most users wont see anything that's not in the theme DB store page.

I would like to be able to "sell" my theme to people, why they should use it, what exactly it does, etc.

I don't really see how a text (maybe a KB) readme would/should be a concern especially when you already have an under 10MB rule to handle if for any reason the read me is too large. Bundling a Description of what a theme does with the theme should be a good thing right?

But with the work around mentioned above i think we both get what we want.

In short what I'm looking to do is describe the theme to users by: 
1. Adding the ability to a description or video or something to the store page for a theme would be nice
2. Adding a Readme to my repo allows me to describe my theme to anyone who visits the github link (including the people here who accept/reject the theme).

**Comment by suchmememanyskill** [comment]
A theme can have a description now in the store page, by adding a `"description": ""` field in the themes.json. Is that not what you're looking for? (see the colored toggles theme as an example)

I am not sure if i want to add videos or multiple images yet on the store

I propose for adding extra files to a repo to make an ignore.json, which deletes all files inside this ignore.json, including itself, when cloning

**Comment by suchmememanyskill** [comment]
Implemented the ignore.json list. Your theme now gets verified correctly
